pkg/machines/libvirt.es6

```diff
@@ -299,7 +313,22 @@ LIBVIRT_PROVIDER = {
             failHandler: buildFailHandler({ dispatch, name, connectionName, message: _("VM SEND Non-Maskable Interrrupt action failed")}),
             args: ['inject-nmi', name]
         });
-    }
+    },
+
+    CHECK_LIBVIRT_STATUS() {
+        logDebug(`${this.name}.CHECK_LIBVIRT_STATUS`);
+        return dispatch => {
+            return cockpit.spawn(['systemctl', '--property=ActiveState', 'show', 'libvirtd'])
```

Comment:
Is `systemctl` available on all supported systems?

Reply:
should be (on their supported versions) 
http://cockpit-project.org/running.html

Reply:
systemctl itself is, but less sure about `--property`. But `systemctl is-active --quiet myunit` has been supported for a long time, and is even easier to use as you just need to look at the exit code. Alternatively, use the [D-Bus API](https://www.freedesktop.org/wiki/Software/systemd/dbus/) (`GetUnit()` and then its `ActiveState` property).

Reply:
I looked at the history of systemd man pages and it should be ok for our versions. I will leave it like it is for now. + we are using the string state in the ui so it is not a big difference